hasura/metadata/databases/default/tables/public_locked_token_distributions.yaml

```diff
@@ -0,0 +1,53 @@
+table:
+  name: locked_token_distributions
+  schema: public
+object_relationships:
+  - name: epoch
+    using:
+      foreign_key_constraint_on: epoch_id
+insert_permissions:
+  - role: user
+    permission:
+      check:
+        epoch:
+          circle:
+            users:
+              profile:
+                id:
+                  _eq: X-Hasura-User-Id
```

Comment:
The insert and update checks should also validate that the user is a circle admin (role = 1)